Police-Data-Accessibility-Project/design-system
# refactor, test: form and input components

- [x]  refactor `Form` and `Input` components (simplify)
- [x]  write tests to cover `Form` and `Input`
Resolves #17 

## To test

- Check out PR, run `npm run test` in the terminal, observe successful test run and the coverage report printed to the console.
- If you want, you can symlink locally and test the UI (I made some updates to form validation and error state  that I think are pretty swanky).
- We're still missing a couple of lines of coverage in `Form`, but none of those lines represent major functionality, so I'm okay with returning to them later as long as everyone else is.
